src/execution/compiler/operator/aggregate_translator.cpp

```diff
@@ -333,7 +338,7 @@ void AggregateTopTranslator::Abort(FunctionBuilder *builder) {
 }
 
 ast::Expr *AggregateTopTranslator::GetChildOutput(uint32_t child_idx, uint32_t attr_idx, terrier::type::TypeId type) {
-  return bottom_->GetOutput(attr_idx);
+  return bottom_->GetOutput(attr_idx + child_idx * bottom_->num_group_by_terms_);
```

Comment:
Why does this need to change?

Reply:
The optimizer passes in a `child_idx` of `1` for aggregate expressions, but the aggregate nodes only have one child. This expression creates an unambiguous attribute index.